vmdb/app/views/container/explorer.html.haml

```diff
@@ -1,6 +1,4 @@
-- content_for :trees do
-  = render(:partial=>"layouts/x_adv_searchbox",
-    :locals=>{:nameonly=>[:containers_tree, :containers_filters_tree].include?(x_active_tree)})
+- content_for :trees do 
   = render(:partial=>'layouts/quick_search')
```

Comment:
@screeley44 i don't think you need to render quick_search view, if you are removing search from Containers Explorer